great_expectations/cli/batch_request.py

```diff
@@ -28,6 +28,11 @@
     sqlalchemy = None
     Inspector = None
 
+DEFAULT_DATA_CONNECTOR_NAMES = [
+    "default_runtime_data_connector_name",
+    "default_inferred_data_connector_name",
+]
+
```

Comment:
Since this is a constant that should only be read from and not written to, can we mark it as such? As of https://www.python.org/dev/peps/pep-0591/, I believe the way to do that is the `Final` annotation from `typing`.


```suggestion
DEFAULT_DATA_CONNECTOR_NAMES: Final[List[str]] = [
    "default_runtime_data_connector_name",
    "default_inferred_data_connector_name",
]

```

Not strictly necessary since we know all caps means a constant (but trying to slowly migrate more of the codebase to strict typing!)